src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java

```diff
@@ -138,6 +145,124 @@ public static DatasetModel populateDatasetModelFromDataset(
+                                  .id(100)
+                                  .name("Condition")
+                                  .count(100)
+                                  .hasChildren(true)),
+                      new SnapshotBuilderDomainOption()
+                          .id(11)
+                          .category("Procedure")
+                          .conceptCount(22500)
+                          .participantCount(11328)
+                          .root(
+                              new SnapshotBuilderConcept()
+                                  .id(200)
+                                  .name("Procedure")
+                                  .count(100)
+                                  .hasChildren(true)),
+                      new SnapshotBuilderDomainOption()
+                          .id(12)
+                          .category("Observation")
+                          .conceptCount(12300)
+                          .participantCount(23223)
+                          .root(
+                              new SnapshotBuilderConcept()
+                                  .id(300)
+                                  .name("Observation")
+                                  .count(100)
+                                  .hasChildren(true))))
+              .programDataOptions(
+                  List.of(
+                      new SnapshotBuilderProgramDataOption()
+                          .id(1)
+                          .name("Year of birth")
+                          .kind(SnapshotBuilderProgramDataOption.KindEnum.RANGE)
+                          .min(1900)
+                          .max(2023),
+                      new SnapshotBuilderProgramDataOption()
+                          .id(2)
+                          .name("Ethnicity")
+                          .kind(SnapshotBuilderProgramDataOption.KindEnum.LIST)
+                          .values(
+                              List.of(
+                                  new SnapshotBuilderListOption().name("Hispanic or Latino").id(20),
+                                  new SnapshotBuilderListOption()
+                                      .name("Not Hispanic or Latino")
+                                      .id(21),
+                                  new SnapshotBuilderListOption()
+                                      .name("No Matching Concept")
+                                      .id(0))),
+                      new SnapshotBuilderProgramDataOption()
+                          .id(3)
+                          .name("Gender identity")
+                          .kind(SnapshotBuilderProgramDataOption.KindEnum.LIST)
+                          .values(
+                              List.of(
+                                  new SnapshotBuilderListOption().name("FEMALE").id(22),
+                                  new SnapshotBuilderListOption().name("MALE").id(23),
+                                  new SnapshotBuilderListOption().name("NON BINARY").id(24),
```

Comment:
(This adds some new options, but I got a little weirded out by only having binary options before so I wanted to add them here)

Reply:
This is fine although the Gender concepts I had set up are the ones in OMOP. Unfortunately OMOP hasn't yet updated the gender domain to include other options, so all non-binary cases fall into concept ID 0. https://www.ohdsi.org/web/wiki/doku.php?id=documentation:vocabulary:gender